### PR TITLE
SCC-4049: Swapping in new Banner component

### PR DESCRIPTION
**Pull request by 7emansell**
Updates to DS version 3.1.0, and replaces the fines notification with the new `Banner` component.

### CODE REVIEW

File: pages/account/[[...index]].tsx [hunk 1]
```diff
@@ -198,27 +198,27 @@ export async function getServerSideProps({ req }) {
   // Parsing path from url to pass to ProfileTabs.
   const tabsPath = req.url.split("/", -1)[2] || null
   const id = patronTokenResponse.decodedPatron.sub
-  // try {
-  const { checkouts, holds, patron, fines } = await MyAccountFactory(id)
-  // Redirecting /fines if user has none.
-  if (tabsPath === "overdues" && fines.total === 0) {
+  try {
```
Comment (7emansell) [inline pages/account/[[...index]].tsx:201]: This is not related to this PR, just error handling i had in `main`

File: src/components/MyAccount/FeesBanner.tsx [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
 
 const FeesBanner = () => {
   return (
-    <Notification
-      notificationType="warning"
-      showIcon={false}
-      className={styles.feesNotification}
-      notificationContent={
+    <Banner
+      sx={{ "> div ": { maxWidth: "unset" }, marginBottom: "m" }}
```
Comment (EdwinGuzman) [inline src/components/MyAccount/FeesBanner.tsx:7]: Why did you have to remove the margins?

File: src/components/MyAccount/FeesBanner.tsx [hunk 3]
```diff
@@ -1,28 +1,25 @@
...
-          <Icon size="medium" name="errorOutline" iconRotation="rotate180" />{" "}
-          <Text>
+          <Text sx={{ marginBottom: 0 }}>
             You have outstanding fees. Borrowing privileges will be suspended
             for cardholders with replacement fees totaling $100 or more. <br />{" "}
             Fees can be paid at any New York Public Library branch in cash, U.S.
             Postal money order, personal check, or{" "}
             <Link
               href={`${appConfig.urls.circulatingCatalog}?openAccount=fines-and-fees`}
+              sx={{
```
[the remark this answers is not in the document]
Comment (EdwinGuzman) [reply src/components/MyAccount/FeesBanner.tsx:18]: We brought this up to Marty and he said that the text color is correct. So you don't need to make them the default blue link color https://github.com/NYPL/nypl-design-system/pull/1533#discussion_r1501091700

File: src/components/MyAccount/FeesBanner.tsx (hunk 2 above)
Comment (7emansell) [reply src/components/MyAccount/FeesBanner.tsx:7]: Design doesn't have them

Comment (bigfishdesign13) [reply src/components/MyAccount/FeesBanner.tsx:7]: I will speak with Apoorva about this.  Those designs are not using the actual banner component, so the max-width is not built-in.  But is should be.

**Review by EdwinGuzman — COMMENTED** [review]
The link color update looks good but are we waiting to confirm the default margin styles?

**Comment by 7emansell** [comment]
> The link color update looks good but are we waiting to confirm the default margin styles?

they're confirmed (https://www.figma.com/file/k2E6Fc3bFV2GebWwCkmJQD?type=design&node-id=2835-56709&mode=design#770382658), just waiting until I can test research catalog again

**Review by dgcohen — APPROVED** [review]

**Review by EdwinGuzman — APPROVED** [review]
Some coordination might be needed for this and #177

**Pull request merged**